src/blueapi/service/main.py

```diff
@@ -108,6 +108,8 @@ def get_app(config: ApplicationConfig):
     dependencies = []
     if config.oidc:
         dependencies.append(Depends(verify_access_token(config.oidc)))
+    else:
+        dependencies.append(Depends(OAuth2(auto_error=False)))
```

Comment:
```suggestion
        dependencies.append(Depends(HTTPBearer(auto_error=False)))
```

I think this would have the same effect in terms of adding the security context to the generated schema and doesn't need the diamond configuration (or dummy values) hard-coded into the schema generation.